src/Microsoft.IdentityModel.Logging/TextWriterEventListener.cs

```diff
@@ -57,7 +58,7 @@ public TextWriterEventListener()
             }
             catch (Exception ex)
             {
-                LogHelper.LogException<InvalidOperationException>(ex, LogMessages.MIML11001);
+                LogHelper.LogExceptionMessage(new InvalidOperationException(LogMessages.MIML11001, ex));
                 throw ex;
```

Comment:
just curious, shouldn't this be just 
```csharp
LogHelper.LogExceptionMessage(new InvalidOperationException(LogMessages.MIML11001, ex));
throw
```

Doesn't `throw ex` changes the StackTrace?